packages/rangelink-vscode-extension/scripts/test-release-run.sh

```
@@ -139,9 +144,16 @@ echo ""
 
+if [[ "$WITH_EXTENSIONS" == "true" ]]; then
+  node "$SCRIPT_DIR/setup-integration-test-settings.js" --suffix -with-ext
+fi
+
 TEST_EXIT=0
 # shellcheck disable=SC2086
-MOCHA_GREP="$GREP_PATTERN" npx vscode-test $VSCODE_TEST_CONFIG 2>&1 | sed 's/\x1b\[[0-9;]*m//g' | tee -a "$REPORT_FILE" || TEST_EXIT=$?
+if [[ -n "$GREP_PATTERN" ]]; then
+  export MOCHA_GREP="$GREP_PATTERN"
+fi
+npx vscode-test $VSCODE_TEST_CONFIG 2>&1 | sed 's/\x1b\[[0-9;]*m//g' | tee -a "$REPORT_FILE" || TEST_EXIT=$?
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)